_posts/2019-05-23-assessing-rails-upgrades.markdown

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Is this Rails Upgrade Project a good fit?"
```

Comment:
@ercohen14 The capitalization of these letters looks a little odd... It should be consistent with past title capitalizations. https://www.ombulabs.com/blog 

Other potential titles: 

- Are you Ready for a Rails Upgrade?
- Is This Rails Upgrade Project a Good Fit? 
- Is your Application Ready for a Rails Upgrade?